docs/root/configuration/other_protocols/thrift_filters/router_filter.rst

```diff
@@ -47,6 +47,7 @@ Since these stats utilize the underlying cluster scope, we prefix with the ``thr
   thrift.upstream_rq_time, Histogram, total rq time from rq complete to resp complete; includes oneway messages.
   thrift.upstream_rq_size, Histogram, Request message size in bytes per upstream
   thrift.upstream_resp_size, Histogram, Response message size in bytes per upstream
+  thrift.cx_drain_close, Counter, Total upstream connections that were closed due to draining.
```

Comment:
should this be thrift.upstream_cx_drain_close to match pattern of other metrics?

Reply:
Yes it should, updated!